galette/lib/Galette/Controllers/DynamicTranslationsController.php

```diff
@@ -171,8 +173,10 @@ public function doDynamicTranslations(Request $request, Response $response): Res
     {
         $post = $request->getParsedBody();
         $post['text_orig'] = htmlspecialchars($post['text_orig'], ENT_QUOTES);
+        $redirect_url = $post['redirect_uri'];
```

Comment:
What if the `$post['redirect_uri']` does not exists?

```suggestion
        if(isset($post['redirect_uri'])) {
            $redirect_url = $post['redirect_uri'];
            unnset($post['redurrect_uri']);
        } else {
            $redirect_url = $this->router->pathFor(
                'dynamicTranslations',
                ['text_orig' => $post['text_orig']]
            )
        }
```

And then later use `$redirect_uri` in response.